astropy/stats/sigma_clipping.py

```diff
@@ -6,6 +6,8 @@
 import warnings
 from ..utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
 
+from scipy.stats import sigmaclip
```

Comment:
This needs to be bracketed with a try/except, since scipy is not a hard requirement.  This will also have to raise the appropriate errors when trying to use this feature when scipy isn't enabled.  Search the codebase for the string "HAS_SCIPY" or "HAVE_SCIPY" to see existing examples of this.